### PR TITLE
Fix #96 and #97

### LINKED ISSUE #96: Slider crashes when nextProps.null !==  null and this.props.step === null.
This issue happens because the function getPrecision is reading this.props which is not up to date yet.
To reproduce the error, Render slider with step={null} then re-render with step={1}

### PR DESCRIPTION
**Pull request by benjycui**

### CODE REVIEW

File: src/Slider.jsx [hunk 1]
```diff
@@ -48,8 +48,7 @@ class Slider extends React.Component {
     if (props.range && upperBound === lowerBound) {
       if (lowerBound === max) {
         recent = 'lowerBound';
-      }
-      if (upperBound === min) {
+      } else {
         recent = 'upperBound';
       }
```
Comment (just-boris) [inline src/Slider.jsx]: maybe

```js
recent = lowerBound === max ? 'lowerBound' : 'upperBound';
```

would be better?

**Comment by just-boris** [comment]
Very good.

**Comment by benjycui** [comment]
Updated.

**Comment by just-boris** [comment]
👍 good job, @benjycui

**Pull request merged**